### PR TITLE
Home page update: remove FAP box, fix mobile padding

### PR DESCRIPTION
**Pull request by rachelcope**
Reason for Change
===================
* https://trello.com/c/vGjpPBDL/259-remove-fap-box-on-homepage-and-any-other-homepages
* https://trello.com/c/geFoOWoa/253-mobile-layout-on-staging

Changes
=======
* Remove FAP box
* Fix mobile spacing

image

### CODE REVIEW

File: app/views/pages/index.html.erb [hunk 1]
```diff
@@ -17,7 +17,7 @@
   </div>
 </div>
 
-<div class="slab slab--white" id="apply-for-programs">
+<!-- <div class="slab slab--white" id="apply-for-programs">
   <div class="card--narrow">
     <%= render 'shared/flashes' %>
```
Comment (jayroh) [inline app/views/pages/index.html.erb]: Looks like this line (displaying the flash messages) should be visible somewhere in order for the tests to pass.

Comment (rachelcope) [reply app/views/pages/index.html.erb]: gotcha, I'll fix that 👌

**Comment by zspencer** [comment]
I brought back the flash! screen shot 2017-09-11 at 1 44 45 pm

**Comment by rachelcope** [comment]
@zspencer okay looks good! That's a better place for the flash anyway

File: app/views/pages/index.html.erb (hunk 1 above)
Comment (zspencer) [reply app/views/pages/index.html.erb]: Pulled the flash up to the top of the screen.

**Pull request merged**